src/main.py

```diff
@@ -87,20 +80,35 @@
 silent = cmdopts.silent
 debug = cmdopts.debug
 
+config_exists = alexapi.config.filename is not None
```

Comment:
I don't think we need this. If there is no config, it fails in alexapi.config in `__init__.py`.